### PR TITLE
Update wake_on_lan.markdown

### PR DESCRIPTION
**Pull request by pixeldoc2000**
## Proposed change

Add Windows Example with IOT Link and MQTT.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/wake_on_lan.markdown [hunk 1]
```diff
@@ -128,3 +128,21 @@ switch:
...
+    name: "TARGET"
+    ...
+    turn_off:
+      service: mqtt.publish
+      data:
+        topic: iotlink/workgroup/computername/commands/shutdown
+```
+**Note**: Replace `workgroup` and `computername`.
+
+See [IOT Link Wiki](https://gitlab.com/iotlink/iotlink/wikis) for further MQTT Commands like Suspend, Hibernate and much more.
```
Comment (frenck) [inline source/_integrations/wake_on_lan.markdown:148]: This is a nice example, but not related to the wake_on_lan integration. This part belongs the documentation of IoT link.

**Comment by frenck** [comment]
Closing this PR because of the above reasoning.

Nevertheless, thanks for willing to contribute, that is always appreciated 👍

File: source/_integrations/wake_on_lan.markdown (hunk 1 above)
Comment (pixeldoc2000) [reply source/_integrations/wake_on_lan.markdown:148]: Hm, thats kind of sad, but it's you decision.

There is already one "real-life examples of how to use the turn_off variable" for Suspending Linux via SSH.

I don't quiet understand why this does not qualify as an other "real-life Example".

Comment (frenck) [reply source/_integrations/wake_on_lan.markdown:148]: Sure, I can make many more. There is one that is more universal for the platforms Home Assistant supports. Another example with natively provided integration of Home Assistant would be nice as well. 

This example, however, is completely external, which is not a concern of this project.

**Pull request closed without merging**